modules/modal/react/lib/ModalContent.tsx

```diff
@@ -80,6 +80,9 @@ const Container = styled('div')({
   background: 'rgba(0,0,0,0.65)',
   animationName: `${fadeIn}`,
   animationDuration: '0.3s',
+  '.wd-no-animation &': {
+    animation: 'none',
+  },
```

Comment:
Do we want any documentation or comments on this like in Popup? This seems like it would be very hard to discover. Or is it meant to be a backdoor that few know?

Reply:
It is definitely one of those backdoor things. I added a comment above just like in Popup